File: ClaimManagementApi/Models/Claim.cs
namespace ClaimManagementApi.Models
{
    public class Claim
    {
        public int Id { get; set; }
        public string Title { get; set; } = string.Empty;

        public string Description { get; set; } = string.Empty;

        public decimal Amount { get; set; }

        public string Status { get; set; } = "Pending";

        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}

File: claim-management-web/src/types/claim.ts
export type Claim = {
  id: number;
  title: string;
  description: string;
  amount: number;
  status: string;
  createdAt: string;
};

export type CreateClaimRequest = {
  title: string;
  description: string;
  amount: number;
};

export type CreateClaimFormData = {
  title: string;
  description: string;
  amount: string;
};

export type UpdateClaimRequest = {
  title: string;
  description: string;
  amount: number;
  status: string;
};

export type UpdateClaimFormData = {
  title: string;
  description: string;
  amount: string;
  status: string;
};
